rts/motoko-rts/src/memory/ic.rs

```diff
@@ -24,13 +24,18 @@ pub(crate) static mut LAST_HP: u32 = 0;
 
 // Provided by generated code
 extern "C" {
-    pub(crate) fn get_heap_base() -> u32;
+    fn get_heap_base() -> u32;
```

Comment:
@osa1 which macro generates this function? Maybe we ca generate `get_aligned_heap_base` as well?

Reply:
This function is generated by the codegen.